lib/client/email/email.js

```diff
@@ -1,6 +1,8 @@
+require('module-alias/register')
```

Comment:
We couldn't require this once somewhere up the chain and have it available to all files?

Reply:
We can at runtime (there is one entrypoint) but it was a problem for the unit tests. Let me just check whether we have to put the `require` in the spec _and_ the file being tested

Reply:
I can see now why I chose to do this:

Ideally, as you say, we should put the call to `module-alias/register` in the _entry_ file, but (obviously) the tests don't use that entry file -- in which case, it should be possible to treat the _spec_ as the entry file, and put the call there, such that the code under test will get the `module-alias` behaviour it expects.

* First problem: `proxyquire` is one the packages we use in the tests and it _sometimes_ interferes with `module-alias` (when `module-alias` is _only_ in the spec)
* Second problem: I'm not 100% certain that there is just one entry point in the application
* Third problem: Cormac's IDE issues

Considering the first and second points, which I was thinking about when I made the changes: I decided it was _safest_ to implement `module-alias` in each file across the project where the behaviour was expected (so, both application and spec files, _if they expected the behaviour_).

Considering the third point, of which Alice reminded me before the stand-up of the morning I made the changes: if no proposed solution to Cormac's IDE issues works for him, then this story and this branch is dead.

On whether this change is beneficial if it's so easily killed: it is. Node module resolution is a notorious problem. I usually get around it by using Babel, which has better tools, but introducing Babel is a) a bigger task, and b) perhaps more of a challenge for you chaps.

On whether I should have ensured there was a solution to Cormac's concerns before I implemented the change: you know, I've been coming out of refinement and story planning sessions in such a state of ... emotional arousal ... and _that_, coupled with my mental health problems (which can affect my short term memory) meant that I plain forgot.

On whether this branch is stale: it isn't. I've updated it a couple of times, and I've mentioned at stand-up that I want to speak with Cormac before making a decision.

If we can't introduce `module-alias` because it breaks someone's workflow then I'll find a way to introduce the behaviour when I _need_ it, as a temporary change, and discard it afterwards. I'm happy to do that, but I'd rather see aliasing in the project than not because relative paths are horrible and confusing in small projects, let alone one of this size.

I'm also happy to write in ES6/7 with Babel and skip over `module-alias` but that's a personal preference and one that I can't justify at this point.